frontend/src/Interfaces/Schedule/Schema/schedule.ts

```
@@ -0,0 +1,50 @@
+
+export type UpdateScheduleRequest = {
+	scheduleId: string
+	slotId?: string
+	accountId?: string
+} //204
+
+// Update returns just status 204, no body
+export type UpdateScheduleResponse = void
+
+// Delete returns just status, no body
+export type DeleteScheduleResponse = void
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)